main/FatesInterfaceMod.F90

```diff
@@ -774,8 +774,9 @@ subroutine SetFatesGlobalElements1(use_fates,surf_numpft,surf_numcft,param_reade
       integer,                    intent(in) :: surf_numpft  ! Number of PFTs in surface dataset
       integer,                    intent(in) :: surf_numcft  ! Number of CFTs in surface dataset
       class(fates_param_reader_type), intent(in) :: param_reader ! HLM-provided param file reader
-  
       integer :: fates_numpft  ! Number of PFTs tracked in FATES
+
+      logical, parameter :: preserve_b4b = .true.
```

Comment:
Do you want to keep this in for a future update?

Reply:
I wasn't sure when CTSM PR: [2898](https://github.com/ESCOMP/CTSM/pull/2898) would go through, but since its through, we can now remove this

Reply:
actually, lets keep this in there and implement the change in a separate PR after this one goes through. This will affect changes in results because we will have different (correct) patch counts.  I'd rather have this PR b4b

Reply:
Sounds good :+1: